proto/config.proto

```diff
@@ -174,6 +174,9 @@ message FrontendConfig {
 
   // The Content Security Policy nonce to use for inline <style> elements.
   string csp_nonce = 57;
+
+  // Whether the community link is enabled in the UI.
+  bool commmunity_link_enabled = 58;
```

Comment:
`mmm` => `mm`

also `link` => `links` since this enables all links

```suggestion
  bool community_links_enabled = 58;
```

Reply:
Dyslexia 😵‍💫
Fixed.